docs/topic/access-creds/cloud-auth.md

```diff
@@ -36,13 +36,14 @@ We have two ways to access AWS accounts.
 
 There are three units of organization in AWS that are relevant to 2i2c.
 
-AWS Accounts
+#### AWS Accounts
 : Collections of services and infrastructure that generated their own bills. Kind-of like `projects` in Google Cloud Platform. For example, the Kubernetes cluster we run for `uwhackweeks` runs in an Account dedicated for this.
```

Comment:
Does turning these into headings work given that I think this is using myst's definition syntax? https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#definition-lists

Reply:
No they don't :woman_facepalming:, I reverted back to using the myst definitions instead, thanks!